src/app.js

```diff
@@ -1 +1,61 @@
-console.log('Hallo daar!');
\ No newline at end of file
+import axios from "axios";
+
+console.log('Hallo daar!');
+
+async function fetchData() {
+  try {
+    const result = await axios.get('https://restcountries.com/v2/all');
+    console.log(result.data);
+    return result.data;
+  } catch (e) {
+    console.error(e);
+  }
+}
+
+async function showData() {
+  let data = await fetchData();
+  data.sort((a, b) => {
+    return a.population - b.population;
+  })
+  data.map((country) => {
```

Comment:
Hier zou je nog spreading kunnen toepassen, zodat je niet overal `country.` voor hoeft te zetten
```suggestion
  data.map(({ name, population, region, flags: { svg } }) => {
```